torchrl/objectives/td3.py

```diff
@@ -313,16 +316,24 @@ def in_keys(self):
     def in_keys(self, values):
         self._in_keys = values
 
+    @property
+    @cache_values
+    def __detach_qvalue_network_params(self):
+        return self.qvalue_network_params.detach()
+
+    @property
+    @cache_values
+    def __stack_actor_params(self):
+        return torch.stack(
+            [self.actor_network_params, self.target_actor_network_params], 0
+        )
```

Comment:
In a function like this, what is the benefit of caching? 
Is it just not to redo the stack operation or am I missing smth?

Reply:
Same as before + lazy stacked construction is reaaaally long so we must cache it if we can (have a look at tensordict benchmarks and introduciton of caches https://pytorch-labs.github.io/tensordict/dev/bench/)